eng/pipelines/global-build.yml

```
@@ -127,11 +127,9 @@ jobs:
 #
 - template: /eng/pipelines/common/platform-matrix.yml
   parameters:
-    jobTemplate: /eng/pipelines/common/global-build-job.yml
-    buildConfig: release
+    jobTemplate: /eng/common/templates/job/source-build.yml
     platforms:
-    - SourceBuild_Linux_x64
+    - Linux_x64
```

```suggestion
    - SourceBuild_Linux_x64
```
We want this to use the `SourceBuild_Linux_x64` platform so it builds using the same Clang and CMake that RedHat will use for their source-build construction.